### DISCUSSION TITLE
Joystick command assignment

### DISCUSSION THREAD

**Discussion by ipran96** [magknight/787aviatorsTracker#823] [Completed things] [answered]
Would be great if MG787 made specific command for joystick assignment , example for fuel control cut - off switch

**Comment by aeroniemi** [comment]
What exactly do you want to be added? It's not too much effort to do it, but if there are others it'd be worth doing them all together

**Comment by ipran96** [comment]
i want  to assign the fuel control switches "run" and "cut off" to a joystick button Unfortunately I can't find any  specific command for it.

**Reply by aeroniemi** [reply]
so you want commands that:
set ENG 1 FUEL CONTROL to RUN
set ENG 1 FUEL CONTROL to CUTOFF
set ENG 2 FUEL CONTROL to RUN
set ENG 2 FUEL CONTROL to CUTOFF

anything else?

**Comment by ipran96** [comment]
ill add my wish here , wish i could Find command for it
1.  A/P toggle button
2. A/T  toggle arm button
3. F/D toggle button
4. A/T toggle button
5. Rnav toggle  button
6. Vnav  toggle button
7. A/P engaged / disengaged toggle button
8. V/S rotary button

**Reply by aeroniemi** [reply]
Most of those already exist in the form of the default commands - the only ones that don't are AT arm

**Comment by aeroniemi** [comment, marked as answer]
the original commands have been added in 1.8.0-rc1

**Reply by ipran96** [reply]
Thankyouuuu

**Reply by VitorAls** [reply]
Hello, I saw that they added the button for fuel control, wouldn't it be possible to add it to the Speed Brakes? I've tried to configure my joystick but it doesn't work. I could assign a button on my Joystick to this function too.

**Reply by aeroniemi** [reply]
> Hello, I saw that they added the button for fuel control, wouldn't it be possible to add it to the Speed Brakes? I've tried to configure my joystick but it doesn't work. I could assign a button on my Joystick to this function too.

are you trying to add it to an axis?

**Reply by VitorAls** [reply]
It's not a normal fixed button, actually inside X-Plane I've already added it and it works with all other aircraft, except for the 787
![B789 - 2022-05-05 07 22 42](https://user-images.githubusercontent.com/104434415/166905128-6bc051fb-0423-481c-8e5a-e72964da5ec8.png)

**Reply by aeroniemi** [reply]
That should work just fine with the 787, although the behavior is probably different due to the way that speedbrake arming is managed
